src/libsystemd/sd-bus/sd-bus.c

```diff
@@ -3138,9 +3137,8 @@ static int bus_exit_now(sd_bus *bus, sd_event *event) {
 
         if (event)
                 return sd_event_exit(event, EXIT_FAILURE);
-        else
-                exit(EXIT_FAILURE);
 
+        exit(EXIT_FAILURE);
         assert_not_reached();
```

Comment:
Not a big matter, but why the assertion exists here?

Reply:
I'd say that this is an unrelated matter. No need to block this PR.

Reply:
Yeah, so it is OK to merge this as is.